From a92060e93e6048d6624efe2301cc8d214bd0ccbf Mon Sep 17 00:00:00 2001
From: Wei Liu <weix.w.liu@intel.com>
Date: Wed, 24 Oct 2018 10:25:38 +0800
Subject: [PATCH 2228/2328] hyper-dmabuf: disable hyper-dmabuf on arch x86

Since hyper-dmabuf enabled treat warnings message as error.
And the warnings will be generated when make target as i386.
This patch will disable hyer-dmabuf on x86 to fix hyper-dmabuf warnings.

Signed-off-by: Wei Liu <weix.w.liu@intel.com>
Acked-by: Kim, Dongwon <dongwon.kim@intel.com>
---
 drivers/dma-buf/hyper_dmabuf/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma-buf/hyper_dmabuf/Kconfig b/drivers/dma-buf/hyper_dmabuf/Kconfig
index 1d91a11..26c5344 100644
--- a/drivers/dma-buf/hyper_dmabuf/Kconfig
+++ b/drivers/dma-buf/hyper_dmabuf/Kconfig
@@ -3,7 +3,7 @@ menu "hyper_dmabuf options"
 config HYPER_DMABUF
 	bool "Enables hyper dmabuf driver"
 	default y
-	depends on (X86=y || X86_64=y)
+	depends on X86_64
 
 choice
 	prompt "Hypervisor"
-- 
2.7.4

